themes/custom/droopler_theme/droopler_theme.theme

```diff
@@ -321,58 +321,6 @@ function droopler_theme_preprocess_field(&$variables) {
   }
 }
 
-/**
```

Comment:
@mariusz-andrzejewski-droptica  tam powyżej też mamy takiego małego IFka którego [pamiętam] dodawałem razem z tym poniżej :) może to usuńmy ?